030-ConversationalAI/Coach/Readme.md

```diff
@@ -0,0 +1,29 @@
+
+# What The Hack - Conversational AI Attach Play
+
+## Introduction
+Welcome to the coach's guide for Conversational AI Attach Play What The Hack. For each challenges we have a solution tips and notes for you coach. You can get an end Bot Composer solution in this folder under Bot Composer Project. You can open that Bot Composer Project on your local Bot Framework Composer application to check the sample core logic this Bot has.
+
+Also, we have prepared a presentation ppt for you to kick off the event under the Coach/Solutions folder. In the same folder, we have provided a high-level architectural diagram Visio file for this solution.
```

Comment:
Please link to the presentation ppt and to the visio file so that people can click from here and download them.

Reply:
When linking to Office documents (PPTX, DOCX, VISX, etc)...  use Github "raw" links so that the link triggers the browser to download the file directly instead of bringing the user to the repo first.

For example... for the FSIBotAttachPlayProgram.pptx, the raw link is: https://github.com/Microsoft-US-OCP-Conversational-AI/WhatTheHack/blob/master/030-ConversationalAI/Coach/Solutions/FSIBotAttachPlayProgram.pptx?raw=true

The "gotcha" with using a raw link is that it's "hard coded" to your fork.  You will need to "predict" the actual raw link after you do the PR to the WTH repo by changing the repo name in the link.   In the example above, you would need to change the link to point to "github.com/microsoft/...":

https://github.com/microsoft/WhatTheHack/blob/master/030-ConversationalAI/Coach/Solutions/FSIBotAttachPlayProgram.pptx?raw=true

Two more gotchas to be aware of with raw links:

1) Raw links are branch specific.  You are doing a PR from the "master" branch of your fork.  The eventual destination will be the "master" branch of the WTH repo.  If you copied the raw link from a different branch, you'd have to change that branch name to "master".    "...WhatTheHack/blob/<branchname>/030-ConverationalAI..."
2) Once you set the raw link to the final destination, it won't work for you test because the files have not been published to that location!  (It won't work until the PR has been approved.)  

I think I need to capture this comment as an update for the WTH author's guide!